cli/src/lib/website/deploySC.ts

```diff
@@ -0,0 +1,22 @@
+import { Provider, SmartContract, U32 } from '@massalabs/massa-web3'
+import { readFileSync } from 'fs'
+import { storageCostForEntry } from '../utils/storage'
+
+const byteCode = readFileSync('src/lib/website/sc/main.wasm')
+const ownerKey = 'OWNER'
+
+export async function deploySC(provider: Provider): Promise<SmartContract> {
+  return SmartContract.deploy(provider, byteCode, undefined, {
+    coins: deployCost(provider),
+  })
+}
+
+export function deployCost(provider: Provider): bigint {
+  const ownerKeyCost = storageCostForEntry(
+    BigInt(ownerKey.length),
+    BigInt(provider.address.length)
+  )
+  const fileListCost = storageCostForEntry(1n, BigInt(U32.SIZE_BYTE))
```

Comment:
Just to keep in mind that we will need to update this when pr of sc will be merge

Reply:
Context, the PR in question is the one about immutability and deweb version: https://github.com/massalabs/DeWeb/pull/126/files#diff-3ebdaef67413baee44de673e3d6d885fe674d291758dbb9fb9857e27aa54e0f1R28